src/dropin-model.js

```diff
@@ -78,6 +78,10 @@ DropinModel.prototype._shouldEmitRequestableEvent = function (options) {
 
 DropinModel.prototype.setPaymentMethodRequestable = function (options) {
   var shouldEmitEvent = this._shouldEmitRequestableEvent(options);
+  var paymentMethodRequestableResponse = {
+    isPaymentMethodSelected: Boolean(options.selectedPaymentMethod),
```

Comment:
I have a slight preference for `paymentMethodIsSelected` so that the if block reads:

```js
if (event.paymentMethodIsSelected) {
```

@intelliot do you have thoughts about that?